From a354b4ecf2434f2f6460b33031aeaf646edf5e64 Mon Sep 17 00:00:00 2001
From: Brad Nelson <bradnelson@chromium.org>
Date: Thu, 4 Jan 2018 00:36:18 -0800
Subject: [PATCH] Disable SharedArrayBuffer by default.

BUG=chromium:798864
R=jschuh@chromium.org,binji@chromium.org

Change-Id: I5ebfae41a4b4c7e89faf071e6d009ea3d9ca30d4
Reviewed-on: https://chromium-review.googlesource.com/849429
Reviewed-by: Ben Smith <binji@chromium.org>
Reviewed-by: Justin Schuh <jschuh@chromium.org>
Reviewed-by: Darin Fisher <darin@chromium.org>
Commit-Queue: Brad Nelson <bradnelson@chromium.org>
Cr-Commit-Position: refs/heads/master@{#527460}
---
 content/public/common/content_features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/content/public/common/content_features.cc b/content/public/common/content_features.cc
index 847a1af6db7b..9075bd853d0f 100644
--- a/content/public/common/content_features.cc
+++ b/content/public/common/content_features.cc
@@ -312,7 +312,7 @@ const base::Feature kServiceWorkerScriptFullCodeCache{
 
 // http://tc39.github.io/ecmascript_sharedmem/shmem.html
 const base::Feature kSharedArrayBuffer{"SharedArrayBuffer",
-                                       base::FEATURE_ENABLED_BY_DEFAULT};
+                                       base::FEATURE_DISABLED_BY_DEFAULT};
 
 // An experiment to require process isolation for the sign-in origin,
 // https://accounts.google.com.  Launch bug: https://crbug.com/739418.
-- 
2.15.1

